src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+
+    @BeforeAll
+    public static void setupGlobalResources() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        user = new User("123456", "123456", 18);
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_validUser_ok() {
+        registrationService.register(user);
+        User stored = Storage.people.get(0);
+        assertNotNull(stored, "User should be stored in Storage");
+    }
+
+    @Test
+    public void register_multipleValidUsers_ok() {
+        User user2 = new User("secondLogin", "secondPassword", 31);
```

Comment:
redundant test 
You are testing RegistrationService, there is no any logic that check count of valid user for example etc